src/libcollections/binary_heap.rs

```diff
@@ -218,10 +219,14 @@ pub struct BinaryHeap<T> {
     data: Vec<T>,
 }
 
-/// A container object that represents the result of the [`peek_mut`] method
-/// on `BinaryHeap`. See its documentation for details.
+/// Object representing a mutable reference to the greatest item on a
```

Comment:
I think we try to avoid using the term 'object' to describe structures since Rust isn't really OO. Personally I use 'structure', though there might be a better word.